capnpc/rust.capnp

```diff
@@ -27,6 +27,32 @@ annotation parentModule @0xabee386cd1450364 (file) :Text;
 #    }
 #  }
 
+annotation imports @0xc3b9fe42a83105cd (file) :List(Import);
+# Allows specifying that the generated code for an import is located in another
+# crate.
+#
+# You only need this if your code uses imported types whose generated code is
+# in another crate. You can only use this annotation once in all the files
+# compiled together.
```

Comment:
Did you consider making this an option to `capnp::CompilerCommand` rather than an annotation? That might make some of this a bit cleaner.

Reply:
Great idea!